includes/ajax-handlers/class-wc-stripe-checkout-sessions-ajax-handler.php

```
@@ -15,6 +15,41 @@ class WC_Stripe_Checkout_Sessions_Ajax_Handler {
+			}
+
+			$session_id = isset( $_POST['session_id'] ) ? sanitize_text_field( wp_unslash( $_POST['session_id'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			if ( empty( $session_id ) ) {
+				throw new Exception( __( 'Missing checkout session ID.', 'woocommerce-gateway-stripe' ) );
+			}
+
+			$checkout_session = WC_Stripe_API::retrieve( "checkout/sessions/{$session_id}" );
+
+			if ( is_wp_error( $checkout_session ) || ! is_object( $checkout_session ) || ! empty( $checkout_session->error ) ) {
+				$message = is_object( $checkout_session ) && ! empty( $checkout_session->error->message ) ? $checkout_session->error->message : __( 'Failed to retrieve checkout session.', 'woocommerce-gateway-stripe' );
+				throw new Exception( $message );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '45', column '15'). (undefined)

(MissingImport)